sw/tools/parrot/disco.py

```diff
@@ -51,5 +51,4 @@ def parse_extra_args(self, args):
 if __name__ == "__main__":
     disco = Disco()
     disco.parse_args()
-    exit(0)
-
+    exit(0)
```

Comment:
a bunch of files have lost their newline at end of file, any reason for that ?

Reply:
Except for a setting mistake swapping between PC's and someones different IDE or editors no.. so will re-add